src/pivotal-ui/components/grids/grids.scss

```diff
@@ -2264,3 +2264,525 @@ page](/containers.html).
+    Long content<br>
+    Long content<br>
+    Long content<br>
+    Long content<br>
+   </div>
+   <div class="panel-footer">
+    Footer
+   </div>
+  </div>
+ </div>
+ <div class="fitem-xs-6 fcol">
+  <div class="fitem-xs fcol panel panel-basic-accent panel-accent-npm-warm-3">
+   <div class="panel-header">
+    <h5 class="panel-title-alt">Medium Panel</h5>
+   </div>
+   <div class="panel-body fitem-xs">
+    Medium content<br>
+    Medium content<br>
+    Medium content<br>
+   </div>
+   <div class="panel-footer">
+    Footer
+   </div>
+ </div>
+</div>
+```
+
+Grid sizes
+
+```html_example
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-1 bg-neutral-10">
+  1
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-2 bg-neutral-10">
+  2
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-3 bg-neutral-10">
+  3
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-4 bg-neutral-10">
+  4
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-5 bg-neutral-10">
+  5
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-6 bg-neutral-10">
+  6
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-7 bg-neutral-10">
+  7
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-8 bg-neutral-10">
+  8
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-9 bg-neutral-10">
+  9
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-10 bg-neutral-10">
+  10
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-11 bg-neutral-10">
+  11
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-12 bg-neutral-10">
+  12
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-13 bg-neutral-10">
+  13
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-14 bg-neutral-10">
+  14
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-15 bg-neutral-10">
+  15
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-16 bg-neutral-10">
+  16
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-17 bg-neutral-10">
+  17
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-18 bg-neutral-10">
+  18
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-19 bg-neutral-10">
+  19
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-20 bg-neutral-10">
+  20
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-21 bg-neutral-10">
+  21
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-22 bg-neutral-10">
+  22
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-23 bg-neutral-10">
+  23
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-24 bg-neutral-10">
+  24
+ </div>
+</div>
+```
+
+Row-major:
+
+```html_example
+<div class="fcol">
+ <div class="fitem-xs frow mvxl">
```

Comment:
what are these things fitem, frow, fcol? Naming is hard... I'm not immediately understanding from the code what I'm looking at... It looks like a fcol is a row? and an fitem is a col? This seems confusing compared to how our current grids are named.